### PR TITLE
Comp 2 - build a form

### PR DESCRIPTION
**Pull request by Fosome**
screen shot 2015-07-27 at 3 05 15 pm

### CODE REVIEW

File: comp2.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
```
Comment (averyvery) [inline comp2.html:4]: It looks like you're mixing/matching indent depth here — should just be two spaces.

File: comp2.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title></title>
+        <meta name="description" content="">
```
Comment (averyvery) [inline comp2.html:7]: Sometimes we put #TODO in content here, just so it's easier to find/fix later.

File: comp2.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+            <form class="route-form">
+              <div class="route-form-header">
+                <div class="route-form-title">
+                  [HQ] <span class="black">&harr;</span> Meadow Event Park
+                </div>
+
+                <ul class="route-form-actions">
+                  <li class="route-form-action">
+                    <a href="#" class="btn-link btn-small btn-secondary">Cancel</a>
```
Comment (averyvery) [inline comp2.html]: This should probably also be a button (regardless of how it's styled), since it's not pointing to a URL.

**Comment by averyvery** [comment]
This looks good — is the design going to be all desktop-sized for now, or are we eventually going to build in mobile-size styles?

File: comp2.html (hunk 1 above)
Comment (Fosome) [inline comp2.html:4]: Sure, I should clean up this boilerplate I grabbed.

File: comp2.html (hunk 3 above)
Comment (Fosome) [inline comp2.html]: Here I would disagree -- it's supposed to be a link, just doesn't have anything to point to yet (no dynamic content).  But it's very much a link to another resources via GET, probably the resource's show page.

File: sass/_content.scss [hunk 4]
```diff
@@ -41,3 +34,110 @@
...
+    transition: background 0.2s ease-in-out;
+  }
+
+  input:focus {
+    background: $white;
+    border: 1px solid $lime-wash;
+    outline: none;
+  }
+
+  select[multiple=true] {
```
Comment (Fosome) [inline sass/_content.scss:137]: Is this bogus?  Did I gobus?

File: comp2.html (hunk 3 above)
Comment (averyvery) [inline comp2.html]: oh, point taken. in that case, maybe "#TODO" the href so it doesn't look like it's just supposed to be a dead link :+1:

Comment (Fosome) [inline comp2.html]: Oh ya, good deal.

File: sass/_content.scss (hunk 4 above)
Comment (averyvery) [inline sass/_content.scss:137]: This should work just fine.

Comment (averyvery) [inline sass/_content.scss:137]: You could probably also have a focus style here.

File: sass/_content.scss [hunk 5]
```diff
@@ -41,3 +34,110 @@
...
+  input {
+    border: 1px solid $grey-wash-hover;
+    background: $grey-wash;
+    color: $black;
+    font-size: 18px;
+    padding: 8px;
+    transition: background 0.2s ease-in-out;
+  }
+
+  input:focus {
```
Comment (averyvery) [inline sass/_content.scss]: One nice trick in sass is to use the ampersand parent selector:

```
input {
  ...styles;

  &:focus {
    ...more-styles;
  }
}
```

**Comment by averyvery** [comment]
:+1:

**Comment by Fosome** [comment]
@averyvery I'm just approaching a desktop buildout for now.  It does _some_ flowing for different screen sizes thanks to flexbox, but not intentional.

File: sass/_content.scss (hunk 5 above)
Comment (Fosome) [inline sass/_content.scss]: That's why it didn't work!  Before I had `:focus` and nothing.  Thanks.

Comment (averyvery) [inline sass/_content.scss]: Ah yeah, there are some tricks to it.

```
input {
  :focus {}
}

compiles to -> input :focus {}, which isn't useful.

input {
  &:focus {}
}

compiles to -> input:focus {}

input {
  .foo &:focus {}
}

compiles to -> .foo input:focus {}
```

**Pull request closed without merging**